pkgs/sage-conf/_sage_conf/_conf.py.in

```diff
@@ -9,6 +9,12 @@ VERSION = "@PACKAGE_VERSION@"
 SAGE_LOCAL = "@prefix@"
 SAGE_ROOT = "@SAGE_ROOT@"
 
+# The semicolon-separated list of GAP root paths. This is the list of
+# locations that are searched for GAP packages. We always prepend
+# SAGE_LOCAL/lib and SAGE_SHARE to this list so it should be empty
+# when the system copy of GAP is used.
+GAP_ROOT_PATHS = "@GAP_ROOT_PATHS@"
```

Comment:
You mean when the spkg version of gap is used?

Reply:
Yes, thanks, fixed.